converter/src/main/java/gov/cms/qpp/conversion/model/error/Detail.java

```diff
@@ -31,42 +39,51 @@ public Detail() {
 	 * Copy constructor
 	 */
 	public Detail(Detail detail) {
-		this(detail.getMessage(), detail.getPath(), detail.getValue(), detail.getType());
+		setErrorCode(detail.getErrorCode());
+		setMessage(detail.getMessage());
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove this call from a constructor to the overridable "setMessage" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1699)